tests/test_utils.py

```diff
@@ -263,14 +263,14 @@ def test_get_item(self):
                                      "hash_key": "travis-test@platform-hca.iam.gserviceaccount.com",
                                      "body": "owner",
                                      "string_set": ["GMAW", "FCAW", "GTAW"],
-                                     "list": [{"S": "Cookies"}, {"S": "Coffee"}, {"N", "3.14159"}]}
+                                     "list": ["Cookies", "Coffee", "3.14159"]}
```

Comment:
Good call!

Reply:
yeah its way cleaner, idk why ddb does not have an option to omit the attribute type....